OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs

```diff
@@ -211,7 +243,7 @@ public bool HandleMouseInput(MouseInput mi)
 					editorWidget.SetTooltip(null);
 
 					// Delete actor.
-					if (underCursor != null && underCursor != Selection.Actor)
+					if (underCursor != null && underCursor != Selection.Actor && !mi.Modifiers.HasModifier(Modifiers.Shift))
```

Comment:
this doesn't fix the bug. Holding shift is only necessary to start dragging, but once you start it can be released

Reply:
Oh hell, you're right